### DISCUSSION TITLE
Could not find org.korge.korlibs.klock:klock:5.1.0

### DISCUSSION THREAD

**Discussion by lemkoleg** [korlibs/korge#2060] [Ideas and discussions]
Libraries such as klock, korim, korio, krypto do not have an update to version 5.x.x. They are not available in the repositories. Now they cannot be connected to the project, separately?

**Comment by soywiz** [comment]
Not right now. There is a current effort on restoring them:

* https://github.com/korlibs/korge/issues/2040
* https://github.com/korlibs/korge/issues/2046
* `korlibs-datastructure` : https://discord.com/channels/728582275884908604/747702355809796156/1183517356119560233

But likely with new artifact names.
Probably something like: `korlibs-time` instead of `klock`, etc.
